docs/_includes/attr-element.adoc

```diff
@@ -4,7 +4,151 @@ Included in:
 - user-manual: Attributes: Setting attributes on an element
 ////
 
+
+////
+
+...there's still some confusion between document attributes and element attributes. We should make this distinction very clear.
+
+In general, Asciidoctor uses attributes as a way to store and deliver metadata. Document attributes are available everywhere and are used to control behavior of the document (they have a broad scope). Element attributes are specifically for controlling an individual element and are scoped to that element. Unlike document attributes, element attributes cannot be referenced using an attribute reference (though, that could change in UniDoc).
+
+Element here refers to a node or a directive. A node is a paragraph, a delimited block, a block macro, an inline macro or a span of formatted text. (maybe that needs to go in the glossary).
+////
+
+// tag::revised[]
+
+
+Element attributes are defined in "attribute lists" between square brackets, and apply only to the immediately following "element",
+where an "element" can be:
+
+* a paragraph
+* a delimited block
+* a block macro
+* an inline macro
+* a span of formatted text
+
+The syntax for setting element attributes is different from that for document attributes, and there is shorthand available for the most common attributes.
+
+For example, in:
+
+----
+[WARNING]
+Keep hands away from moving machinery
+----
+
+* The attribute list contains a single attribute `style` (a "positional attribute"), which has the value `WARNING`. 
```

Comment:
This attribute is also known as the block name. However, I think the right way to explain it is that the positional attribute is the style, and one of the aspects of the style is the block name (but may also include id, role and options). Maybe "style" isn't generic enough of a name for this group, though. Perhaps "signature"?